index.html

```diff
@@ -31,21 +31,22 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
     <link href="assets/site.css" rel="stylesheet">
     <link href="assets/globalsite.css" rel="stylesheet">
-    <link href="assets/social-share-kit.css" rel="stylesheet"/>
+    <link href="assets/social-share-kit.css" rel="stylesheet">
     <link href="assets/hlidac.css" rel="stylesheet">
     <script src="assets/vuejs-datepicker-patched.min.js"></script>
+    <script src="assets/vuejs-auto-complete-0.9.0-patched-min.js"></script>
     <script src="assets/lang-cs.js"></script>
-    <script src="assets/citycouncils.min.js"></script>
+    <script src="assets/citycouncils.min.js"></script>  <!-- seznam obecnich uradu, ale nejsou tam vsechny -->
+    <script src="assets/cityzipname.min.js"></script>   <!-- seznam psc + obci (snad vsech) pro naseptavac -->
     <script src="https://cdn.jsdelivr.net/npm/moment@2.24.0/moment.min.js" identity="sha384-QliZbpszoo4jiVyo529dYiKgGQySM0I/kSJcvQcVkMXCqiIoEfRQLsvUwVNLcOqP" crossorigin="anonymous"></script>
     <script src="https://cdn.jsdelivr.net/npm/vue@2.6.8/dist/vue.min.js" integrity="sha384-tJs0vlPyuG7XcysSkZ0VtWsM/uztJkUtpLwb6HzV5NSgSwGqdW61IBOBJDhbUf+K" crossorigin="anonymous"></script>
     <script src="https://cdn.jsdelivr.net/npm/vue-router@3.0.2/dist/vue-router.min.js" integrity="sha384-EnzMJi5PirMz1dgUr9a4DUHwg7e71+fxk3jLLfJn1vljGydxX8c6bdDYdPDE8224" crossorigin="anonymous"></script>
     <script src="https://raw.githack.com/eKoopmans/html2pdf/master/dist/html2pdf.bundle.min.js" integrity="sha384-Ho2Qx5q7Y92BZSxEZtDh2+hUqcudf/uXp77PK0XXawmB+tmreY4nUttlKWXnUWtn" crossorigin="anonymous"></script>
-    <script src="https://cdn.jsdelivr.net/npm/vuejs-auto-complete@0.9.0/dist/build.js" integrity="sha384-asU05qXSwk3ZdJx7bfEM0Iau0NvJdny68xiGvriuI3Y1AtQfRRNd9Mo4IApTgejh" crossorigin="anonymous"></script>
     <!-- Global site tag (gtag.js) - Google Analytics -->
     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-154075-24"></script>
     <script>
         window.dataLayer = window.dataLayer || [];
-        function gtag(){dataLayer.push(arguments);}
+        function gtag() { dataLayer.push(arguments); }
```

Comment:
tady poprosím zbytečně neměnit historii editace řádků jen kvůli whitespace

Reply:
Tohle mi bohužel dělá Visual Studio. Snažím se to vracet, ale někdy to uteče.